src/uct/ib/ud/verbs/ud_verbs.c

```diff
@@ -535,7 +539,7 @@ uct_ud_verbs_iface_query(uct_iface_h tl_iface, uct_iface_attr_t *iface_attr)
         return status;
     }
 
-    iface_attr->overhead = 105e-9; /* Software overhead */
+    iface_attr->overhead = UCT_UD_VERBS_IFACE_OVERHEAD; /* Software overhead */
```

Comment:
we need to take  it from config value (in case it is modified for some platform)